src/osmedile/intellij/stringmanip/swap/SwapCharacterToFromIntAction.java

```diff
@@ -0,0 +1,158 @@
+package osmedile.intellij.stringmanip.swap;
+
+import com.intellij.openapi.actionSystem.DataContext;
+import com.intellij.openapi.editor.Editor;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.text.StringEscapeUtils;
+import osmedile.intellij.stringmanip.AbstractStringManipAction;
+import osmedile.intellij.stringmanip.CharacterSwitchingSettings;
+import osmedile.intellij.stringmanip.config.PluginPersistentStateComponent;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SwapCharacterToFromIntAction extends AbstractStringManipAction<Object> {
+    private static final Pattern isChar = Pattern.compile("(?s)'(.*)'");
+    private static final Pattern isNumber = Pattern.compile("\\d+");
+
+    private CharacterSwitchingSettings.Encoding forceEncoding = null;
+
+    @Override
+    protected String transformSelection(Editor editor, Map<String, Object> actionContext, DataContext dataContext, String s, Object additionalParam) {
+        if (StringUtils.isEmpty(s)) {
+            return s;
+        }
+        Matcher m;
+        if ((m = isChar.matcher(s)).matches()) {
+            return toDigits(m.group(1));
+        } else if (isNumber.matcher(s).matches()) {
+            return toChar(s);
+        }
+//        char i = '\f';
```

Comment:
Tinker/Debug artifact accidentally left in place. This artifact was used while testing to verify the switch was working within the editor once the plugin was updated.